stix_shifter_modules/reversinglabs/stix_translation/query_constructor.py

```diff
@@ -19,9 +19,27 @@
 
 class QueryStringPatternTranslator:
     # Change comparator values to match with supported data source operators
+    comparator_lookup = {
```

Comment:
This shouldn't be here. All operators should be defined in the `operators.json` file.